frontend/ui/widget/textboxwidget.lua

```diff
@@ -1168,6 +1174,28 @@ function TextBoxWidget:update(scheduled_update)
     self.scheduled_update = nil
 end
 
+function TextBoxWidget:setText(text)
+    if text == self.text then
+        return
+    end
+
+    self.text = text
+
+    self:_computeTextDimensions()
+
+    self:update()
+
+    -- Don't break the reference
```

Comment:
At least, please remove these blank lines, that's not in the style of the rest of the module.